ddtrace/internal/context_manager.py

```diff
@@ -101,7 +103,4 @@ def reset(self):
         _DD_CONTEXTVAR.set(None)
 
 
-if CONTEXTVARS_IS_AVAILABLE:
-    DefaultContextManager = ContextVarContextManager
-else:
-    DefaultContextManager = ThreadLocalContext
```

Comment:
are we able to remove `ThreadLocalContext` too? seems like it isn't being used here anymore? (other than maybe not breaking API if someone is using manually?

Reply:
Can't because of the API. I'm thinking of adding a deprecation warning to be removed in the next release however.

Reply:
Added